tests/user.test.js

```
@@ -25,7 +25,7 @@ describe('User', () => {
     try {
       await sequelize.sync({ force: true });
     } catch (err) {
-      console.log(err);
+      // Error occurred
```

You silently swallow any error thrown by `sequelize.sync({ force: true })` here. If DB sync fails in CI or locally you won't get diagnostic info. Consider logging the error (or rethrowing) so failures are visible during test runs. See the test setup in this file for context .